libs/hdf-converters/src/dbprotect-mapper.ts

```diff
@@ -99,53 +103,53 @@ function idToString(id: unknown): string {
 }
 
 export class DBProtectMapper extends BaseConverter {
-  mappings: MappedTransform<ExecJSON.Execution, ILookupPath> = {
+  withRaw: boolean;
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
     platform: {
       name: 'Heimdall Tools',
-      release: HeimdallToolsVersion,
-      target_id: ''
+      release: HeimdallToolsVersion
```

Comment:
I'm not sure if it would be an easy find and replace or more complicated, but should we change this to whatever it is now? Because Heimdall Tools is deprecated?

Reply:
[Are you referring to this](https://github.com/mitre/heimdall2/issues/4523)? If so, it should be a simple fix on this mapper since it's just naming.